packages/nuxt/src/app/composables/manifest.ts

```diff
@@ -0,0 +1,18 @@
+import { joinURL } from 'ufo'
+import { useRuntimeConfig } from '#app'
+
+export interface NuxtAppManifest {
+  id: string
+  timestamp: number
+  routeRules: Record<string, any>
+  prerendered: string[]
+}
+
+let manifest: Promise<NuxtAppManifest>
+
+export function getAppManifest (): Promise<NuxtAppManifest> {
+  const config = useRuntimeConfig()
+  // TODO: use build id injected
+  manifest = manifest || $fetch(joinURL(config.app.cdnURL || config.app.baseURL, '_builds/latest.json'))
```

Comment:
It would be nice if we can handle promise rejections to clear value and improve error

Reply:
PS: We need a ttl somehow. User opens page and fetches manifest > a deploy happens > client side navigation is again not aware of updates!

Reply:
excellent point. on it 👍

Reply:
actually, seems I already implemented at https://github.com/nuxt/nuxt/commit/4f85cff8d94c575c5f4238a4a8f8472beb1bce5d. Review very appreciated 🙏